doc/monitoring/metrics_reference.rst

```diff
@@ -710,3 +710,61 @@ Event loop tx thread information:
             -   Event loop prolog time (ms)
         *   -   ``tnt_ev_loop_epilog_time``
             -   Event loop epilog time (ms)
+
+Synchro
+-------
+
+Show the current state of a synchronous replication.
+
+..  _metrics-reference-synchro:
+
+..  container:: table
+    ..  list-table::
+        :widths: 25 75
+        :header-rows: 0
+
+        *   -   ``tnt_synchro_queue_owner``
+            -   Instance ID of the current synchronous replication master.
+
+        *   -   ``tnt_synchro_queue_term``
+            -   Current queue term.
+
+        *   -   ``tnt_synchro_queue_len``
+            -   How many transactions are collecting confirmations now.
+
+        *   -   ``tnt_synchro_queue_busy``
+            -   Whether the queue is processing any system entry (CONFIRM/ROLLBACK/PROMOTE/DEMOTE).
+
+        *   -   ``tnt_synchro_quorum``
+            -   Current value of ``box.cfg.replication_synchro_quorum``
+
+Election
+--------
+
+Show the current state of a replica set node in regards to leader election.
+
+..  _metrics-reference-election:
+
+..  container:: table
+    ..  list-table::
+        :widths: 25 75
+        :header-rows: 0
+        *   -   ``tnt_election_state``
+            -   election state (mode) of the node.
+                When election is enabled, the node is writable only in the leader state.
+                Possible values:
```

Comment:
Well, Prometheus, InfluxDB and Graphite (and most, if not every, metric storages) do not support string metrics. Moreover, such format breaks Prometheus metrics collect (InfluxDB should just skip a single invalid metrics, but it still will make it useless).